packages/components/src/tabs/styles.module.scss

```diff
@@ -71,37 +66,41 @@ export const StyledTabList = styled( Ariakit.TabList )`
 			width: calc( var( --antialiasing-factor ) * 1px );
 			transform: translateX( calc( var( --selected-start ) * var( --direction-factor ) * 1px ) )
 				scaleX( calc( var( --selected-width, 0 ) / var( --antialiasing-factor ) ) );
-			border-bottom: var( --wp-admin-border-width-focus ) solid ${ COLORS.theme.accent };
+			border-bottom: var( --wp-admin-border-width-focus ) solid theme.$components-color-accent;
```

Comment:
@mirka What should we do about `--wp-admin-border-width-focus` ? It is defined in [the `admin-scheme` mixin](https://github.com/WordPress/gutenberg/blob/502f4a382a5ba02d6a2bd08f4e34651a1dd94392/packages/base-styles/_mixins.scss#L506-L523), and I guess there's no guarantee for it to be defined at all times?

Should we keep using `--wp-admin-border-width-focus` for now  or should we create a local package-wide equivalent? (In any case, we could be able to stop using this variable if the Theme offers this value as a token)